test/test_sync.py

```diff
@@ -326,3 +309,51 @@ def test_sync_failures(mc):
     # check synced images
     assert not os.path.exists(os.path.join(driver_dir, "img1.png"))
     assert os.path.exists(os.path.join(driver_dir, "images", "img2.jpg"))
+
+def test_multiple_tables(mc):
+    project_name = "mediasync_test"
+    full_project_name = API_USER + "/" + project_name
+    work_project_dir = os.path.join(TMP_DIR, project_name + '_work')  # working dir for mediasync
+    driver_dir = os.path.join(TMP_DIR, project_name + '_driver')  # destination dir for 'local' driver
+
+    cleanup(mc, full_project_name, [work_project_dir, driver_dir])
+    prepare_mergin_project(mc, project_name)
+
+    # patch config to fit testing purposes
+    config.update({
+        'ALLOWED_EXTENSIONS': ["png"],
+        'MERGIN__USERNAME': API_USER,
+        'MERGIN__PASSWORD': USER_PWD,
+        'MERGIN__URL': SERVER_URL,
+        'MERGIN__PROJECT_NAME': full_project_name,
+        'PROJECT_WORKING_DIR': work_project_dir,
+        'DRIVER': "local",
+        'LOCAL__DEST': driver_dir,
+        'OPERATION_MODE': "copy",
+        'REFERENCES': [{"file": "survey.gpkg", "table": "notes", "local_path_column": "photo", "driver_path_column": "ext_url"},
+                       {"file": "survey.gpkg", "table": "photos", "local_path_column": "photo", "driver_path_column": "ext_url"}]
+    })
+
+    main()
+    # check synced images
+    assert os.path.exists(os.path.join(driver_dir, "img1.png"))
+
+    # check references have been changed and pushed
+    project_info = mc.project_info(full_project_name)
+    assert project_info["version"] == "v2"
+
+    copied_file = os.path.join(driver_dir, "img1.png")
+    # check that both tables were updated
+    gpkg_conn = sqlite3.connect(os.path.join(work_project_dir, config.references[0].file))
+    gpkg_cur = gpkg_conn.cursor()
+    sql = f"SELECT count(*) FROM {config.references[0].table} WHERE {config.references[0].driver_path_column}='{copied_file}'"
+    gpkg_cur.execute(sql)
+    assert gpkg_cur.fetchone()[0] == 1
+    # check that both tables were updated
+    gpkg_conn = sqlite3.connect(os.path.join(work_project_dir, config.references[1].file))
+    gpkg_cur = gpkg_conn.cursor()
+    sql = f"SELECT count(*) FROM {config.references[1].table} WHERE {config.references[1].driver_path_column}='{copied_file}'"
+    gpkg_cur.execute(sql)
+    assert gpkg_cur.fetchone()[0] == 1
+    shutil.rmtree(work_project_dir)
```

Comment:
I suggest to do cleanup at the beginning of the test, so we have working dir available when test fails to debug